pkg/kubectl/cmd/create_pdb.go

```diff
@@ -60,7 +60,8 @@ func NewCmdCreatePodDisruptionBudget(f cmdutil.Factory, cmdOut io.Writer) *cobra
 	cmdutil.AddValidateFlags(cmd)
 	cmdutil.AddPrinterFlags(cmd)
 	cmdutil.AddGeneratorFlags(cmd, cmdutil.PodDisruptionBudgetV1GeneratorName)
-	cmd.Flags().String("min-available", "1", i18n.T("The minimum number or percentage of available pods this budget requires."))
+	cmd.Flags().String("min-available", "", i18n.T("The minimum number or percentage of available pods this budget requires."))
```

Comment:
Hrm, this is a breaking change for `create pdb`. I think the most correct approach here is to continue defaulting min-available to 1 for the current generator and create a new generator where we default maxUnavailable or leave it up to users.

cc: @kubernetes/sig-cli-api-reviews